README.rst

```diff
@@ -62,33 +62,95 @@ It is suggested to install a pre-commit hook in order to apply Black before push
     $ pre-commit install
 
 
-Project Progres
+Project Progress
 ----------------
 
 View the project Kanban board `here <https://github.com/debrief/pepys-import/projects/3>`_
 
 Setup
 -----
 
-To prepare for running ensure these tools and packages are installed:
+To prepare for running ensure Python 3.7 or later are installed in your system.
+You can check your Python 3 version with the following command::
 
-* Python 3 - at least Python 3.6.4 or later
-* SQL Alchemy 1.3
+    $ python3 --version
 
-For Postgres support and unit tests these packages are also required:
+If you don't have Python 3.7+ in your system, please download it from `python.org <https://www.python.org/downloads/>`_
 
-* geoalchemy2
-* psycopg2
-* nose2 (0.9.1)
+----------
+For Ubuntu
+----------
+
+Installing Spatialite
+*********************
+1. Easy Way
+"""""""""""
+
+Open a terminal and run the following command::
+
+    $ sudo apt install spatialite-bin
+
+2. Alternative Way (Compiling From Source)
+""""""""""""""""""""""""""""""""""""""""""
+
+On Debian-based distributions the following libraries are necessary to run SpatiaLite::
+
+    $ sudo apt install zlib1g-dev libfreexl1
+    $ sudo apt-get install sqlite3 libsqlite3-dev
+    $ sudo apt-get install- y libsqlite3-mod-spatialite
+    $ sudo apt-get install binutils libproj-dev gdal-bin libgeos-dev libxml2-dev
+
+After all libraries are installed, it is necessary to download and run the latest SpatiaLite version::
+
+    $ wget https://www.gaia-gis.it/gaia-sins/libspatialite-sources/libspatialite-X.Y.Z.tar.gz
+    $ tar xaf libspatialite-X.Y.Z.tar.gz
+    $ cd libspatialite-X.Y.Z
+    $ ./configure
+    $ make
+    $ sudo make install
+
+Installing PostGIS
+******************
+
+The best way to install PostGIS is running the codes as follows::
+
+    sudo apt-get install libpq-dev python-dev
+    sudo add-apt-repository ppa:ubuntugis/ubuntugis-unstable
+    sudo apt-get update
+    sudo apt-get install postgis
+
+Cloning the Repository
+**********************
+Navigate to folder you would like to download the repository and run the following command::
+
+    $ git clone https://github.com/debrief/pepys-import.git
+
+Creating Python Environment
+***************************
+Virtual Environment might be used to run the project. For creating a proper one,
+the following commands must be executed respectively in your project directory::
+
+    $ pip install virtualenv
```

Comment:
@BarisSari - is plain `pip` going to install this for Python 2.x?

Reply:
Ok, @BarisSari - on my vanilla Ubuntu VM, I don't have pip installed. So, I guess we need to install pip3, then use pip3 in this call.

Reply:
I changed it with pip3 and add instructions to install pip3 at the beginning:
https://github.com/debrief/pepys-import/pull/66/commits/1609db3ad01e313d86b990f011d09542af803e24